src/main/proto/wfa/measurement/api/v2alpha/exchange_step.proto

```diff
@@ -55,4 +69,42 @@ message ExchangeStep {
     // The step has permanently failed. Terminal state.
     FAILED = 4;
   }
+
+  // Represents how to access an output/input.
+  message SharedData {
+    // Encrypted, serialized `SignedData` messages containing serialized
+    // `Location` payloads.
+    bytes encrypted_signed_location = 1;
+
+    // TODO(@efoxepstein): refactor to use a resource URI or something generic.
+    message Location {
+      // This should correspond to one of the values in the
+      // `ExchangeWorkflow.Step.output_labels` map for a step.
+      string label = 1;
+
+      oneof location {
```

Comment:
interesting... so the system won't actually pass in any blobs but instead send generic location urls to download?

Reply:
Correct.